jenkins_job_wrecker/cli.py

```diff
@@ -54,7 +54,7 @@ def get_xml_root(filename=False, string=False):
 
 
 # Walk an XML ElementTree ("root"), and return a YAML string
-def root_to_yaml(root, name):
+def root_to_yaml(root, name, ignore_action=False):
```

Comment:
What do you think about referring to `action` as `actions` everywhere in the code? In some places it's "action", but it looks like the XML element is "actions"?

Reply:
Agreed